cfpb/cfgov-refresh
# Migrating to webpack 4

Migrating to Webpack 4.  You can read about some of the benefits here: https://medium.com/webpack/webpack-4-released-today-6cdb994702d4


## Changes

- Modified scripts to migrate code to Webpack 4. This PR also includes modifications to allow us to run in dev mode, by setting the NODE_ENV environment variable.  This PR allows each app to have it's own webpack-config.js file.

## Testing
1. Run `frontend.sh`.
2. Run `export NODE_ENV=development` at the command line.
3. Run `gulp scripts` and verify that soucemaps are enabled.
4. Browse the site and ensure all the expected pages are working. Check the console for errors. 

## Screenshots


## Notes

- This isn't DRY but we shall get there.

## Todos

-

## Checklist
